src/ontology/mondo-ingest.Makefile

```diff
@@ -208,7 +209,10 @@ mappings: $(ALL_MAPPINGS)
 mapping-progress-report: unmapped-terms-tables unmapped-terms-docs
 
 .PHONY: unmapped-terms-docs
-unmapped-terms-docs: $(foreach n,$(ALL_COMPONENT_IDS), reports/$(n)_unmapped_terms.tsv)
+unmapped-terms-docs: docs/reports/unmapped.md
+
+# todo: ideally unmapped_docs.py would also take file path(s) as input args. Currently, running `docs/reports/unmapped_%.md` for any source will trigger this for all sources.
+docs/reports/unmapped.md docs/reports/unmapped_%.md: $(foreach n,$(ALL_COMPONENT_IDS), reports/$(n)_unmapped_terms.tsv)
```

Comment:
## `docs/reports/unmapped.md docs/reports/unmapped_%.md`
This was missing, and only the `.PHONY` alias `unmapped-terms-docs` was present. This I guess you could say is an anti-pattern.

I also added a `todo`:
> ideally `unmapped_docs.py` would also take file path(s) as input args. Currently, running `docs/reports/unmapped_%.md` for any source will trigger this for all sources.

Reply:
> Do we need `docs/reports/unmapped_%.md` in isolation? 

Doesn't seem like we need. I think that was the original design decision but it may have been my personal implementation decision.

> we could call this goal `just docs/reports/unmapped.md` - even if it creates other files as well..

I don't mind calling it that if we change it to only output that, but I think we've been trying to be comprehensive in making sure all our outputs are accounted for in make goal targets. However it sounds like you are not preferring that in this (and possibly other/) situations? Maybe because it makes the targets / makefile hard to read?

Reply:
> this is similar to the... other thread. I am still worried that... will cause... to run twice. I am inclined to trust you...

Looks like you mean [this thread](https://github.com/monarch-initiative/mondo-ingest/pull/550#discussion_r1622995244).

In that thread, I tested using `--dry-run` and found that the execution of both patterns were equivalent, so you don't need to worry.

So in choosing between the 2 patterns, I would say to leave it as-is because it is easier to read.

Reply:
I agree with Nico that this change does run more of a risk of having `docs/reports/unmapped.md docs/reports/unmapped_%.md` run multiple times than what is there currently. However, given the current structure of the file and how goals are run and results of the full build (and my own testing running this goal) this seems to be ok for now.